org.gnomeradio.Radio.json

```diff
@@ -0,0 +1,56 @@
+{
+  "app-id" : "org.gnomeradio.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "44",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "rename-icon" : "gnome-radio",
+  "copy-icon" : true,
+  "rename-desktop-file" : "gnome-radio.desktop",
+  "rename-appdata-file" : "gnome-radio.appdata.xml",
+    "finish-args" : [
+     "--share=network",
+     "--socket=session-bus",
```

Comment:
this is not allowed. You need to figure out which DBus name you want to access.
`--log-session-bus` passed to `flatpak run` will help.